File: Q14/Q14.js
function pascalTriangle(rows) 
{
    for (let i = 0; i < rows; i++) 
    {
        let coef = 1;
        let output = " ".repeat(rows - i);
        for (let j = 0; j <= i; j++) {
            output += coef + " ";
            coef = coef * (i - j) / (j + 1);
        }
        console.log(output);
    }
}

pascalTriangle(5);

File: Q14/Q14.py
rows = 5
for i in range(rows):
    print(" " * (rows - i), end="")
    coef = 1
    for j in range(i + 1):
        print(coef, end=" ")
        coef = coef * (i - j) // (j + 1)
    print()
